sg360_backend/csv_data.py

```diff
@@ -0,0 +1,45 @@
+import psycopg2
+import csv
+import dotenv
+import os
+
+
+dotenv.load_dotenv()
+
+# Establish a connection to the PostgreSQL database

```

Comment:
Add a TO-DO comment here stating a .env file must be created, it makes code more readable.